docs/getting-started/install-configure.md

```diff
@@ -375,12 +375,12 @@ detailed instructions.
 ## Uninstall Provider
 ```
 kubectl delete -f provider.yaml
-kubectl delete secret -n crossplane-system --all
 ```
 
 ## Uninstall Crossplane
 ```
 helm delete crossplane --namespace crossplane-system
```

Comment:
```suggestion
# Let's check whether there is any managed resource is up before deleting everything.
kubectl get managed
# If there is any, please delete them first so that you don't lose track of them.
helm delete crossplane --namespace crossplane-system
```

Reply:
good suggestions, updated.